app/src/main/java/com/android/sample/ui/components/AttendantPreview.kt

```diff
@@ -0,0 +1,68 @@
+package com.android.sample.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.PersonRemove
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.android.sample.model.image.ImageViewModel
+import com.android.sample.model.profile.User
+import com.android.sample.resources.C.Tag.LINE_STROKE
+import com.android.sample.resources.C.Tag.MEDIUM_PADDING
+import com.android.sample.resources.C.Tag.SMALL_IMAGE_SIZE
+import com.android.sample.resources.C.Tag.STANDARD_PADDING
+import com.android.sample.ui.camera.ProfileImage
+
+@Composable
+fun AttendantPreview(
```

Comment:
Adding this function as a composables is a very good idea as it could be used in future features